wallet/src/main/java/com/android/identity_credential/wallet/ui/destination/consentprompt/ConsentPromptData.kt

```diff
@@ -0,0 +1,40 @@
+package com.android.identity_credential.wallet.ui.destination.consentprompt
+
+import androidx.compose.runtime.Immutable
+import androidx.compose.runtime.Stable
+import com.android.identity.credential.CredentialRequest
+
+/**
+ * ConsentPromptData is passed to ConsentPrompt to show the appropriate requested credentials.
+ */
+@Stable
+@Immutable
+data class ConsentPromptData(
+    // Object extracted from the document request
+    val credentialRequest: CredentialRequest,
+    // requested doc type
+    val docType: String,
+    // document name of credential being used to respond with requested data
+    val documentName: String,
+    // id of credential that provides documentName - used after Consent Prompt succeeds
+    val credentialId: String,
+    // hard-coded at the moment to be "Verifier"
+    val verifierName: String = "Verifier"
```

Comment:
I think this should be a `verifier: TrustPoint?` and you can dig out the name and icon to show when it's set. If not set the prompt should show _The following information is being requested_.

Since https://github.com/openwallet-foundation-labs/identity-credential/commit/ae70da9eeacc3a703102770e8cd903b8d3d27d9e just landed, I would use `TrustManager` from the caller (e.g. `PresentationActivity`) to see if we can find a `TrustPoint` for the reader.

You want to get the `TrustManager` from `WalletApplication` and initialize it there and at initialization time add a `TrustPoint` for the `owf_identity_credential_reader_cert.pem` file (copy it from from `appholder/src/main/res/raw/owf_identity_credential_reader_cert.pem`). Essentially something like this

```
val owfIdentityCredentialReaderRoot = Certificate.fromPem(String(
  context.resources.openRawResource(
    R.raw.owf_identity_credential_reader_cert
  ).readBytes()))
trustManager = TrustManager()
trustManager.addTrustPoint(TrustPoint(
    owfIdentityCredentialReaderRoot.javaX509Certificate,
    "OWF Identity Credential Reader",
    null // TODO: add image for our own reader
  )
)
```

I realize I'm asking for a little bit of extra work but I think it's worth it and we need to do this _anyway_.

Longer term the trust point might also include a "trust score" so the ConsentPrompt can show the appropriate warnings ("The requesting party is unknown. Be careful about sharing!" and implement appropriate friction.